spec/services/hyrax/admin_set_member_query_service_spec.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.describe Hyrax::AdminSetMemberQueryService, clean_repo: true do
+  let(:blacklight_config) { CatalogController.blacklight_config }
+  let(:repository) { Blacklight::Solr::Repository.new(blacklight_config) }
+  let(:current_ability) { instance_double(Ability, admin?: true) }
+  let(:scope) { double('Scope', can?: true, current_ability: current_ability, repository: repository, blacklight_config: blacklight_config) }
```

Comment:
Why does scope need to respond to `can?`. Shouldn't it be using `current_ability.can?`?

Reply:
I grabbed some code from another spec when I made these, and that was a carryover that I missed removing. It's done now.